.github/workflows/serverless.yml

```diff
@@ -6,13 +6,22 @@ on:
     branches: [master]
   schedule:
     - cron: 0 4 * * *
+  workflow_dispatch:
+    inputs:
+      runNightly:
```

Comment:
Passing explicitly which version to use as latest would make the concurrency group easier. It would also allow running the workflow manually with a specific nightly if needed.